src/command_modules/azure-cli-deploymentmanager/azure_bdist_wheel.py

```diff
@@ -0,0 +1,54 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
```

Comment:
Walk me through this, do you have any particular needs that mean that you don't want to build your wheel from the sdist?

Reply:
Don't have any particular needs and can remove this if the generated wheel is consistent with how it is done for other standard modules. Almost all of the modules I've looked at have this, so wasn't sure if I had to add this.